src/main/java/io/fabric8/maven/docker/util/DockerFileUtil.java

```diff
@@ -119,6 +119,26 @@ public static FixedStringSearchInterpolator createInterpolator(MojoParameters pa
                 .withExpressionMarkers(delimiters[0], delimiters[1]);
     }
 
+    /**
+     * Fetch the location of docker file in case of simgle configuration
+     *
+     * @return directory containing the Dockerfile
+     */
+    public static String getDockerFileDirectory() {
+        if(checkFileExists("./Dockerfile")) {
```

Comment:
I would not rely on the current working directory (which might e.g. completely elsewhere in a multi-module project). Instead you `${project.baseDir}` (or the MavenProject's equivalent) should be taken into account (which is the absolute path to the module base directory).